controllers/chefs/deleteChef/index.js

```diff
@@ -0,0 +1,7 @@
+import { Chef } from '../../../models/chef/index.js';
+
+export const deleteChef = async (req, res) => {
+  await Chef.findByIdAndDelete(req.params.chefId);
+
```

Comment:
You still need to remove the role of the chef from the user document in the roles array

Reply:
Sure, I've fixed  this bug